bindata/network/01-network-configurtion-policy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: {{ .BridgeName }}
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  desiredState:
+    interfaces:
```

Comment:
So instead of prescribing linux bridge I wonder if it might be better to just pass in the entire desiredState/interfaces section via our CRD. Essentially just directly exposing the nmstate CRD's interface via ours? I'm not against using a bridge for now as our default, perhaps we could just have that in our sample config?

Reply:
added it, please check latest push.